### PR TITLE
Save restore create2

### PR DESCRIPTION
**Pull request by sinev-valentine**

### CODE REVIEW

File: runtime/src/eval/mod.rs [hunk 1]
```diff
@@ -2,13 +2,18 @@
 mod macros;
 mod system;
 
-pub use system::save_return_value;
+pub use system::{save_return_value, save_created_address};
 use crate::{Handler, Runtime, ExitReason, ExternalOpcode, CallScheme};
 
+/// ...
```
Comment (s-medvedev) [inline runtime/src/eval/mod.rs:8]: Why do you need this comment? It's very strange to see it in code.

File: runtime/src/eval/system.rs [hunk 2]
```diff
@@ -268,30 +268,10 @@ pub fn create<H: Handler>(
...
-				ExitReason::Fatal(e) => {
-					push!(runtime, H256::default());
-					Control::Exit(e.into())
-				},
-			}
+			save_created_address(runtime, reason, address, return_data, handler)
 		},
 		Capture::Trap(interrupt) => {
-			push!(runtime, H256::default());
+			// push!(runtime, H256::default());
```
Comment (s-medvedev) [inline runtime/src/eval/system.rs:275]: Write a short description of why you commented out this line.

File: runtime/src/eval/mod.rs (hunk 1 above)
Comment (sinev-valentine) [reply runtime/src/eval/mod.rs:8]: this comment is required for the build

File: runtime/src/eval/system.rs (hunk 2 above)
Comment (sinev-valentine) [reply runtime/src/eval/system.rs:275]: This case was not implemented.
Now the case is used to implement the "create" opcode

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**